src/components/VerboseTextFieldArray.tsx

```diff
@@ -72,6 +78,17 @@ export default function VerboseTextFieldArray(
         </Grid>
       </Grid>
 
+      {/* If the whole field is in a state, show a description text here. */}
+      {state ? (
+        <FormHelperText
+          className={`Mui-${state.statusValue}`}
+          sx={statusColors}
+        >
+          {state.statusDescription}
+        </FormHelperText>
+      ) : undefined}
```

Comment:
This may be extracted in a small component if it makes things more readable. Also, I think it is a good practice here to have a consistent type out of the ternary, so I would replace `undefined` by `<></>`, an empty component.

Reply:
Good idea! Used the component for the `VerboseTextField` as well now.
Commit: 15cce17